src/webhooks/src/http-api.js

```diff
@@ -0,0 +1,222 @@
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const bodyParser = require('./body-parser');
+const logger = require('bunyan').createLogger({name: 'http-api'});
+const FlowsDao = require('./flows-dao');
+const assert = require('assert');
+
+const FLOW_ID_PARAM = 'id';
+const WEBHOOK_ROUTE_PATH = `/hook/(:${FLOW_ID_PARAM})(/\\w*)?`;
+const REQUEST_ID_HEADER = 'x-request-id';
+// Delay in msec with error response
+// to prevent try-and-error way to find
+// suitable webhook IDs
+const ANTIHACK_RESPONSE_DELAY = 1000;
+
+function generateRequestId() {
```

Comment:
Generally I'd prefer to move this methods into  class itself.  This gives us 
1. ability to write unit tests for them
2. ability to use them in class ancestors